webpack.config.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path');
```

Comment:
Nice use of Webpack. But let's for the moment stay away from this

Once your code works, we will move to Webpack